### ISSUE TITLE
"info()" Method ("interface.py") Improvement

### ISSUE THREAD

**Issue by lelaus** [newcleo-dev-team/lbh15#53]
- [1] delete `name` variable since used only once;
- [2] delete `rvalue` variable and place `return` within both the if statement branches;

**Comment by panDanieleN** [comment]
Merged into master.

No need to update documentation and tests.
